File: src/main/java/com/p2p/lending/dao/RechargeDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Recharge;




public interface RechargeDao {
	List<Recharge> selectrc(Map<String, Object> map);
	int selectcount(Map<String, Object> map);
	List<Recharge> selectall();
	int sumczmoneyre();
	int sumdzmoneyre();
}

File: src/main/java/com/p2p/lending/dao/RechargeDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Recharge


interface RechargeDao {

    fun selectrc(map: Map<String, Any>): List<Recharge>

    fun selectcount(map: Map<String, Any>): Int

    fun selectall(): List<Recharge>

    fun sumczmoneyre(): Int

    fun sumdzmoneyre(): Int

}
